lesson_03/quiz/src/quizzes/quizzes.module.ts

```diff
@@ -1,14 +1,15 @@
 import { Module } from '@nestjs/common';
+import { AmiyahJonesQuiz } from './amiyah_jones_quiz.js';
 import { AngelicaCQuiz } from './angelica_c_quiz.js';
 import { AnotherQuiz } from './another_quiz.js';
 import { AnthonyMaysQuiz } from './anthony_mays_quiz.js';
-import { XavierCruzQuiz } from './xavier_cruz_quiz.js';
-import { DasiaEnglishQuiz } from './dasia_english_quiz.js';
 import { ChigazoGrahamsQuiz } from './chigazo_graham_quiz.js';
+import { DasiaEnglishQuiz } from './dasia_english_quiz.js';
 import { JosephCaballeroQuiz } from './joseph_caballero_quiz.js';
 import { OyeyemiJimohQuiz } from './oyeyemi_jimoh_quiz.js';
+import { TommyTranQuiz } from './Tommy_Tran_quiz.js';
```

Comment:
Need to change to the lowercase file. I just deleted it for you on your branch, but you might need to `git pull` in order to retore your quiz file.

Reply:
changed to lowercase in the quizzes.module.ts .